modules/renter/contractor/formcontract.go

```diff
@@ -28,34 +28,47 @@ var (
 
 // maxSectors is the estimated maximum number of sectors that the allowance
 // can support.
-func maxSectors(a modules.Allowance, hdb hostDB) (uint64, error) {
-	if a.Hosts == 0 || a.Period == 0 {
+func maxSectors(a modules.Allowance, hdb hostDB, tp transactionPool) (uint64, error) {
+	if a.Hosts <= 0 || a.Period <= 0 {
```

Comment:
it's equivalent either way. But when it's written like this you can be certain it's correct even if you don't know whether the integers are signed or not.